mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -1422,3 +1424,197 @@ def _auto_indent(repr_str, indent):
         repr_str += ' ' * indent + 'prev_voxel_generator=\n'
         repr_str += f'{_auto_indent(repr(self.prev_voxel_generator), 8)})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class AffineResize(object):
+    """get the affine transform matrix to the tagert size."""
+
+    def __init__(self, img_scale=None, down_ratio=None, bbox_clip_border=True):
+
+        self.img_scale = img_scale
+        self.down_ratio = down_ratio
+        self.bbox_clip_border = bbox_clip_border
+
+    def __call__(self, results):
+
+        if 'center' not in results:
+            assert 'size' not in results
+            assert 'affine_aug' not in results
+            img = results['img']
+            height, width = img.shape[:2]
+            center = np.array([width / 2, height / 2], dtype=np.float32)
+            size = np.array([width, height], dtype=np.float32)
+            results['affine_aug'] = False
+        else:
+            img = results['img']
+            assert 'center' in results
+            assert 'size' in results
+            assert 'affine_aug' in results
```

Comment:
Why do we need so many assertions?

Reply:
Acttually it's used to judge whether RandomShiftScale is used before AffineResize, we can discuss more details

Reply:
Then merge these assertions and add a comment before it/add assertion messages

Reply:
Considering the key 'center' has been able to be used to judge whether RandomShiftScale before AffineResize, I remove these redundent assertations and add comments